### PR TITLE
[MODULAR] Guns Galore Rebalance 

### PR DESCRIPTION
**Pull request by MassacreDemigirl**
## About The Pull Request
Makes most of the guns in the Guns Galore to be more in line with rest of the guns in game in weight class (Bulky instead of normal sized items). Makes the PPSh Bulky in order to create a variation between it and the PPS (folding the stock to fit normal size). Also makes the MP40's description in line with Contributor Guidelines. 

## How This Contributes To The Skyrat Roleplay Experience
Removes the backpack M16 and alike, putting them more in line with other guns. Should hopefully make the intentions of players with those guns more visible rather then them taking a rifle out of their backpack.

## Changelog

:cl:
balance: AKM, M16, FG42, PPSh and StG are now bulky instead of normal sized items. Goodbye backpack M16!
fix: Made the MP40's description in line with contributor guidelines
spellcheck: FG-42 is named as is instead of Fallschirmjägergewehr 42
/:cl:

### CODE REVIEW

**Comment by Vizoplez** [comment]
another gun PR

sadge

**Comment by Draganfrukts** [comment]
Based (Saved Me Work)

File: modular_skyrat/modules/gunsgalore/code/guns/mp40/mp40.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/ballistic/automatic/submachine_gun/mp40
 	name = "\improper MP-40"
-	desc = "The instantly recognizable 'nazi gun'. Extremely outdated SMG that has only seen service during Sol-3's second World War. This one's a poor, unlicensed reproduction."
+	desc = "The instantly recognizable 'axis gun'. Extremely outdated SMG that has only seen service during Sol-3's second World War. This one's a poor, unlicensed reproduction."
```
Comment (Draganfrukts) [inline modular_skyrat/modules/gunsgalore/code/guns/mp40/mp40.dm]:
```suggestion
	desc = "The instantly recognizable 'kraut gun'. Extremely outdated SMG that has only seen service during Sol-3's second World War. This one's a poor, unlicensed reproduction."
```
Probably more fitting

**Review by Draganfrukts — COMMENTED** [review]
I DIDN'T MEAN TO MAKE A FUCKING REVIEW

File: modular_skyrat/modules/gunsgalore/code/guns/fg42/fg42.dm [hunk 2]
```diff
@@ -1,12 +1,13 @@
 /obj/item/gun/ballistic/automatic/battle_rifle/fg42
-	name = "\improper Fallschirmjägergewehr 42"
+	name = "\improper FG 42"
```
Comment (Draganfrukts) [inline modular_skyrat/modules/gunsgalore/code/guns/fg42/fg42.dm]:
```suggestion
	name = "\improper FG-42"
```

File: modular_skyrat/modules/gunsgalore/code/guns/fg42/fg42.dm [hunk 3]
```diff
@@ -29,7 +30,7 @@
 	multiple_sprites = AMMO_BOX_FULL_EMPTY
 
 /obj/item/gun/ballistic/automatic/battle_rifle/fg42/modern
-	name = "\improper Fallschirmjägergewehr 42 MK. VII"
+	name = "\improper FG 42 MK. VII"
```
Comment (Draganfrukts) [inline modular_skyrat/modules/gunsgalore/code/guns/fg42/fg42.dm]:
```suggestion
	name = "\improper FG-42 MK. VII"
```

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**